MarkdownView.js

```diff
@@ -59,7 +59,7 @@ const DefaultRules : Rules = Object.freeze(mergeRules(
   ),
   {
     heading: {
-      match: SimpleMarkdown.blockRegex(/^ *(#{1,6}) *([^\n]+?) *#* *(?:\n *)*\n/),
+      match: SimpleMarkdown.blockRegex(/^ *(#\s{1,6}) *([^\n]+?) *#* *(?:\n *)*\n/),
```

Comment:
The added `\s` seems to be incorrect. That will match `# header`, `#  header`, `#   header`, `#    header`, `#     header` and `#      header`.

You'll just want to replace the second asterisks (`*`) in the regex with a `+`, as presently we're looking for zero or more spaces after the 1-6 hashes (`#`), where as we should be looking for one or more spaces.

Reply:
```
match: SimpleMarkdown.blockRegex(
          /^ *(#{1,6})\s *([^\n]+?) *#* *(?:\n *)*\n/
        )
```
The last push I did. Do I need to make a new pull request?

Reply:
Oh, I see what happened. Prettier changed the file. ugh.

Reply:
No need to submit a new pull request.

```
git rebase -i HEAD~2
```

then drop the first commit (`d`) and edit the second one (`e`) to correct the regex.

Once you're done editing:

```
git rebase --continue
git push -f origin master
```